app/components/link-button.tsx

```diff
@@ -0,0 +1,36 @@
+import type { PropsWithChildren } from "react";
+import { Link } from "@remix-run/react";
+import type { RemixLinkProps } from "@remix-run/react/dist/components";
+
+import { classNames } from "~/utils/misc";
+
+type Props = PropsWithChildren<{
+  href: string;
+  variant: "solid" | "outline";
+  prefetch?: RemixLinkProps["prefetch"];
+}>;
+
+// TODO: The red color used in this button is not compliant with WCAG contrast requirements when used for text on our
+//       "almost white" background. Consider using something darker? It's fine as a button bg color.
+export const LinkButton: React.FC<Props> = ({
+  href,
+  variant,
+  prefetch = href.startsWith("/") ? "intent" : undefined,
+  children = href,
+}) => (
+  <Link
+    to={href}
+    prefetch={prefetch}
+    className={classNames(
+      "w-[200px] rounded-md border-2 font-bold min-h-[40px] leading-snug m-1 block py-3 text-center",
```

Comment:
Legge til `transition ease-in-out`
```suggestion
      "w-[200px] rounded-md border-2 font-bold min-h-[40px] leading-snug m-1 block py-3 text-center transition ease-in-out",
```

Jeg ser at reduced motion er allerede håndter i `global.css` på lik måte som gatsby siden, supert.

https://github.com/capraconsulting/nettsiden/blob/main/app/global.css#L71-L79
https://github.com/capraconsulting/capraconsulting.no/blob/master/packages/web/src/styles/global.scss#L92-L100